src/components/Editor/Preview/Popup.js

```diff
@@ -96,8 +100,24 @@ export class Popup extends Component {
   }
 
   renderObjectPreview(expression: string, root: Object) {
+    let expandDepth = 0;
+    const { loadedObjects } = this.props;
+    const getObjectProperties = id => loadedObjects[id];
+    let roots = this.getChildren(root, getObjectProperties);
+
+    if (!roots) {
+      return null;
+    }
+
+    if (isReactComponent(roots)) {
+      roots = roots.filter(r => ["state", "props"].includes(r.name));
+      expandDepth = 1;
```

Comment:
nice... for now though, we can leave that out too. i'm okay w/ users expanding :) 

sometimes you'll want state, sometimes props. Either way, we'll iterate quickly